.gitignore

```diff
@@ -14,4 +14,6 @@ pom.xml.asc
 .idea/
 .cpcache/
 .DS_Store
-.nrepl-port
\ No newline at end of file
+.nrepl-port
+.lsp
+.clj-kondo/
```

Comment:
Should this be `.clj-kondo/.cache/`? So any changes to `.clj-kondo/config.edn` aren't ignored